src/ray/gcs/client_test.cc

```diff
@@ -42,7 +42,12 @@ class TestGcs : public ::testing::Test {
 
   virtual void Stop() = 0;
 
+  int64_t NumCallbacks() { return num_callbacks_; }
```

Comment:
nit: `int64_t num_callbacks() const { return num_callbacks_; }`

smaller case for cheap methods; const when you can

up to you